lib/prism/translation/parser/lexer.rb

```diff
@@ -413,7 +413,7 @@ def to_a
                 while token.type == :STRING_CONTENT
                   current_length += token.value.bytesize
                   # Heredoc interpolation can have multiple STRING_CONTENT nodes on the same line.
-                  is_first_token_on_line = lexed[index - 1] && token.location.start_line != lexed[index - 2][0].location&.start_line
+                  is_first_token_on_line = lexed[index - 1] && token.location.start_line != lexed[index - 2]&.first.location&.start_line
```

Comment:
Is it possible for lexed[index - 2] to be nil here? If it is, then we need &.location as well. If it's not, then we shouldn't use &.first

Reply:
I meant this to be "safe" safe navigation. I don't believe this matters for valid syntax, but with invalid one I can't say so wanted to be on the safe side.